test/test_nameko_socket_server.py

```diff
@@ -23,9 +23,9 @@ def test_server(container_factory):
     container.start()
 
     sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-    sock.connect(('', 6000))
-    sock.sendall('hello\n')
-    data = sock.recv(1024)
+    sock.connect(('127.0.0.1', 6000))
```

Comment:
No, i don't. If the extension to be prepared only for Unix based systems. I tested python 3.x version on Windows, where i need that.. But it's not pivotal change of the PR..